HHS/OPRE-OPS
# 2626 configure demo data

## What changed

- Add instructions for running the application with demo data to the README.
- Create a new demo_data directory with a demo SQL script that:
  - Assign user `521` (Director Dave) as the project officer and user `522` (User Demo) as a team member on Agreements 1, 9, and 10.
  - Updates the status of BLIs on Agreement 10 to PLANNED.
- Create a script to run SQL statements.
- Add a `data-demo` profile and related services to the Docker file.

## Issue

[Configure Demo Data for Application Presentation #2626 ](https://app.zenhub.com/workspaces/opre-ops-62d197b4c468970013bcd2ec/issues/gh/hhs/opre-ops/2626)

## How to test

Use `docker compose --profile data-demo up --build` to run the application locally. Verify that Agreements 1, 9, and 10 are configured appropriately.

## Definition of Done Checklist
<del> - [ ] Automated a11y tests updated and passed
<del> - [ ] 90%+ Code coverage achieved
<del> - [ ] Form validations updated
